gnuradio-runtime/lib/block.cc

```diff
@@ -739,9 +748,8 @@ namespace gr {
         basic_block_sptr blk = global_block_registry.block_lookup(block);
         blk->post(port, pmt::cons(pmt::mp("done"), pmt::mp(true)));
 
-        //std::cout << "notify finished --> ";
+        //GR_LOG_DEBUG (d_debug_logger, "notify finished -->");
```

Comment:
I thought the main advantage of log4cpp is that it can be enabled by the user during runtime, i.e., ones doesn't have to recompile to enabled or disable logging.